tests/components/daydream-controls.test.js

```diff
@@ -158,23 +157,64 @@ suite('daydream-controls', function () {
   });
 
   suite('buttonchanged', function () {
-    test('emits trackpadchanged on buttonchanged', function (done) {
+    test('emits trackpadchanged on buttonchanged for button 0', function (done) {
       var el = this.el;
       var component = el.components['daydream-controls'];
 
       el.sceneEl.systems['tracked-controls'].controllers = component.controllersWhenPresent;
 
       component.checkIfControllerPresent();
 
+      const eventState = {value: 0.5, pressed: true, touched: true};
+
       // Install event handler listening for triggerchanged.
       this.el.addEventListener('trackpadchanged', function (evt) {
-        assert.ok(evt.detail);
+        assert.deepEqual(evt.detail, eventState);
+        done();
+      });
+
+      // Emit buttonchanged for the trackpad button.
+      this.el.emit('buttonchanged', {id: 0, state: eventState});
+    });
+
+    test('emits menuchanged on buttonchanged for button 1', function (done) {
+      var el = this.el;
+      var component = el.components['daydream-controls'];
+
+      el.sceneEl.systems['tracked-controls'].controllers = component.controllersWhenPresent;
+
+      component.checkIfControllerPresent();
+
+      const eventState = {value: 0.5, pressed: true, touched: true};
+
+      // Install event handler listening for triggerchanged.
+      this.el.addEventListener('menuchanged', function (evt) {
```

Comment:
You can just do `el.addEventListener('menuchanged', function (evt) {`

Reply:
Nice, I copied that from another place, so I fixed up all instances.